docs/editor.md

```diff
@@ -33,37 +34,38 @@ const App = () => {
 
 ### `value: Value`
 
-this is the value that the editor should display. You will usually load that from a database.
-It has the type `Value`, but should generally considered to be "opaque", so don't rely on its internal structure.
+This is the content that the editor will display. The data for this prop may come from any source, eg a file or database or API etc. It is a JSON representation of the entire content that should be rendered, including the external visible information, and internal information required to render the information - eg ids, versions, plugin info, etc.
```

Comment:
really nice! Maybe omit or reword the "including the external visible information and internal information" part, personally I don't really get that and i think its more confusing than helpful.

Reply:
I'll try rewording it better in the next version. "external visible information" = what the viewer sees. "internal information" =the meta data that controls the rendering